ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -718,7 +720,7 @@ let on_request :
     | Some handler ->
       Fiber.return
         ( Reply.later (fun send ->
-              let* res = handler ~params state in
+              let* res = handler ~params rpc state in
```

Comment:
Instead of this change, you can just partially apply `rpc` to the requests that actually need this argument.